cropper/src/main/java/com/canhub/cropper/CropImageOptions.kt

```diff
@@ -38,522 +38,312 @@ open class CropImageOptions : Parcelable {
     @JvmField
-    var maxZoom: Int
+    val maxZoom: Int = 4,
 
     /**
      * The initial crop window padding from image borders in percentage of the cropping image
      * dimensions.
      */
     @JvmField
-    var initialCropWindowPaddingRatio: Float
+    val initialCropWindowPaddingRatio: Float = 0.1f,
 
     /** whether the width to height aspect ratio should be maintained or free to change.  */
     @JvmField
-    var fixAspectRatio: Boolean
+    val fixAspectRatio: Boolean = false,
 
     /** the X value of the aspect ratio.  */
     @JvmField
-    var aspectRatioX: Int
+    val aspectRatioX: Int = 1,
 
     /** the Y value of the aspect ratio.  */
     @JvmField
-    var aspectRatioY: Int
+    val aspectRatioY: Int = 1,
 
     /** the thickness of the guidelines lines in pixels. (in pixels)  */
     @JvmField
-    var borderLineThickness: Float
+    val borderLineThickness: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 3f, Resources.getSystem().displayMetrics),
 
     /** the color of the guidelines lines  */
     @JvmField
     @ColorInt
-    var borderLineColor: Int
+    val borderLineColor: Int = Color.argb(170, 255, 255, 255),
 
     /** thickness of the corner line. (in pixels)  */
     @JvmField
-    var borderCornerThickness: Float
+    val borderCornerThickness: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 2f, Resources.getSystem().displayMetrics),
 
     /** the offset of corner line from crop window border. (in pixels)  */
     @JvmField
-    var borderCornerOffset: Float
+    val borderCornerOffset: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 5f, Resources.getSystem().displayMetrics),
 
     /** the length of the corner line away from the corner. (in pixels)  */
     @JvmField
-    var borderCornerLength: Float
+    val borderCornerLength: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 14f, Resources.getSystem().displayMetrics),
 
     /** the color of the corner line  */
     @JvmField
     @ColorInt
-    var borderCornerColor: Int
+    val borderCornerColor: Int = Color.WHITE,
     /**
      * The fill color of circle corner
      */
     @JvmField
-    var circleCornerFillColorHexValue: Int
+    val circleCornerFillColorHexValue: Int = Color.WHITE,
 
     /** the thickness of the guidelines lines. (in pixels)  */
     @JvmField
-    var guidelinesThickness: Float
+    val guidelinesThickness: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 1f, Resources.getSystem().displayMetrics),
 
     /** the color of the guidelines lines  */
     @JvmField
     @ColorInt
-    var guidelinesColor: Int
+    val guidelinesColor: Int = Color.argb(170, 255, 255, 255),
 
     /**
      * the color of the overlay background around the crop window cover the image parts not in the
      * crop window.
      */
     @JvmField
     @ColorInt
-    var backgroundColor: Int
+    val backgroundColor: Int = Color.argb(119, 0, 0, 0),
```

Comment:
Let's please extract this value so we know what colour we expect here. just for reading